platform/engineering/design-docs/2021-01-13-web-component-library.md

```diff
@@ -0,0 +1,139 @@
+# Web Component library
+
+**Author(s):** Brooks Johnson  
+**Last Updated:** January 13, 2021  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Chris Valarida_ \[ \], _Michael Fleet_ \[ \]
+
+
+## Overview
+
+### Objective
+
+The goal is to establish a component library based on Web Components, which are a browser standard. This will supercede the current `formation-react` library and some of the liquid HTML templates currently in `vets-website`. A non-goal is creating an entirely new visual experience as part of the component library.
+
+The intended audience for this document is frontend engineers.
+
+### Background
+
+We currently have some "components" implemented in HTML/CSS (and sometimes JS) in the `formation` npm package. These can be used anywhere where `formation` is loaded. We also have `formation-react`, which is a package with some of those patterns implemented as React components. These can only be used in React apps. Lastly, in `vets-website` we have some Liquid templates which implement some of the components/patterns of the design system, and these are only used for content builds.
```

Comment:
For clarity, prefer bullet points. (Source: [Google Technical Writing One](https://developers.google.com/tech-writing/one/lists-and-tables) course.)

Maybe something like:
```markdown
The problem this design document intends to solve is the re-implementation of common components across different repositories and libraries, using different technologies.

Currently, common components may be found in one or more of three places:
- The `formation` NPM package
  - More information here
- The `component-library` NPM package
  - More information here
- Liquid templates in `vets-website`
  - More information here
```